lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -375,7 +375,7 @@ namespace Js
 
     ScriptContext::~ScriptContext()
     {
-        Assert(isFinalized || !isInitialized);
+        Assert(isFinalized);
```

Comment:
Ah I see. Maybe it's not worth doing that bookkeeping? In which case, deleting this assert seems like the right thing to do I guess.